Makefile

```diff
@@ -94,8 +106,18 @@ test-unit-coverage: test-unit
 	gocov report build/coverage/coverprofile.json
 
 .PHONY: test-e2e
-test-e2e: crds
-	GO111MODULE=on TEST_WATCH_NAMESPACE=${TEST_NAMESPACE} ginkgo ${TEST_E2E_FLAGS} test/e2e
+test-e2e: crds test-e2e-plain
+
+.PHONY: test-e2e-plain
+test-e2e-plain:
+	GO111MODULE=on \
+	TEST_OPERATOR_NAMESPACE=${TEST_NAMESPACE} \
+	TEST_WATCH_NAMESPACE=${TEST_NAMESPACE} \
+	TEST_E2E_OPERATOR=${TEST_E2E_OPERATOR} \
```

Comment:
Too many env variables can be confusing, you could simplify this lines by doing:
```sh
TEST_E2E_OPERATOR=${TEST_E2E_OPERATOR:-start_local}
TEST_E2E_CREATE_GLOBALOBJECTS=${TEST_E2E_CREATE_GLOBALOBJECTS:-true}
....
```

Reply:
This syntax does not seem to work in make.